src/views/assets/Cloud/Strategy/StrategyList.vue

```diff
@@ -52,10 +52,11 @@ export default {
               updateRoute: 'CloudStrategyUpdate',
               hasClone: false,
               canDelete: ({ row }) => {
-                return row.name !== 'default'
+                return this.$hasPerm('xpack.delete_strategy') && row.name !== 'default'
               },
               canUpdate: ({ row }) => {
-                return row.name !== 'default'
+                return this.$hasPerm('xpack.change_strategy') && row.name !== 'default' &&
+                  !this.$store.getters.currentOrgIsRoot
               }
             }
           }
```

Comment:
The only difference I noticed is:

* The first `export default {` statement should be moved to the top of the file.

This is typically done at compile time using webpack loaders (like Babel) to ensure better readability and maintainability. This change would not impact functionality but improves organization slightly.